packages/flutter/lib/src/widgets/table.dart

```diff
@@ -280,7 +280,7 @@ class _TableElement extends RenderObjectElement {
   Table get widget => super.widget as Table;
 
   @override
-  RenderTable get renderObject => super.renderObject as RenderTable;
+  RenderTable get renderObject => super.renderObject! as RenderTable;
```

Comment:
This pattern strikes me as odd. Why is renderObject nullable in the super-class, but not in (many of its) subclasses?

Reply:
`RenderObjectElement.renderObject` is never null once mounted.